File-Formats/PDF/Working-with-OCR/Dot-NET-Framework.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: OCR processor for .NET Framework with tesseract | Syncfusion
```

Comment:
OCR processor for .NET PDF Framework with tesseract | Syncfusion

Reply:
Resolved

Reply:
resolved